src/proxy-ui-api/frontend/tests/e2e/pages/tabs/settingsTab.js

```diff
@@ -25,6 +25,7 @@
  */
 
 const path = require('path');
+var assert = require('assert');
```

Comment:
Should use `const` for immutable variables.

Reply:
Fixed, used `const` instead